public-api/src/main/java/org/pmiops/workbench/service/DomainInfoService.java

```diff
@@ -31,4 +34,25 @@ public DomainInfoService(DomainInfoDao domainInfoDao, DomainMapper domainMapper)
                 .map(domainMapper::dbModelToClient)
                 .collect(Collectors.toList());
     }
+
+    public List<Integer> getTestOrderFilter(TestFilter testFilter, OrderFilter orderFilter) {
+        Integer getTests = null;
+        Integer getOrders = null;
+
+        if (testFilter.equals(TestFilter.SELECTED) && orderFilter.equals(OrderFilter.SELECTED)) {
+            getTests = 1;
```

Comment:
nit: can we create constants for getTests and getOrders for the values 0, 1 and 2. I don't understand what those numbers mean?

Reply:
I have changed this part of the code and added string column in there.